doc/locale/ja/LC_MESSAGES/news/15.po

```diff
@@ -17,12 +17,21 @@ msgstr ""
 msgid "News - 15 series"
 msgstr "お知らせ - 15系"
 
-msgid "Release 15.10 - 2025-05-14"
-msgstr "15.10リリース - 2025-05-14"
+msgid "Release 15.11 - 2025-06-05"
+msgstr "15.11リリース - 2025-06-05"
 
 msgid "Improvements"
 msgstr "改良"
 
+msgid "Added support for MariaDB 10.6.22, 10.11.12 on Ubuntu"
+msgstr "UbuntuのMariaDB 10.6.22, 10.11.12 をサポート"
```

Comment:
The latest package in noble is MariaDB 10.11.13.
```suggestion
msgstr "UbuntuのMariaDB 10.6.22, 10.11.13 をサポート"
```
noble: https://packages.ubuntu.com/noble/mariadb-server
jammy: https://packages.ubuntu.com/jammy/mariadb-server

Reply:
Thank you for  your comment.
I've fixed it.